t/lib/OpenQA/SeleniumTest.pm

```diff
@@ -54,40 +55,11 @@ sub start_app {
     $mojoport = $args->{mojoport} // $ENV{MOJO_PORT} // Mojo::IOLoop::Server->generate_port;
 
     $startingpid = $$;
-    $mojopid     = fork();
-    if ($mojopid == 0) {
-        log_info("inserting fixtures into database\n");
-        if ($schema_hook) {
-            $schema_hook->();
-        }
-        else {
-            OpenQA::Test::Database->new->create;
-        }
-
-        log_info("starting web UI\n");
-        $ENV{MOJO_MODE} = 'test';
-        my $daemon = Mojo::Server::Daemon->new(
-            listen => ["http://127.0.0.1:$mojoport"],
-            silent => 1,
-        );
-        $daemon->build_app('OpenQA::WebAPI');
-        $daemon->run;
-        Devel::Cover::report() if Devel::Cover->can('report');
-        _exit(0);
+    if (!$schema_hook) {
```

Comment:
I believe this check can be moved to Test::Utils::create_webapi, then you don't need to pass hook parameter from first call in create_webapi in 40-script_load_templates.t